crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -867,7 +867,7 @@ bitflags::bitflags! {
         const UNLIT                      = 1 << 5;
         const TWO_COMPONENT_NORMAL_MAP   = 1 << 6;
         const FLIP_NORMAL_MAP_Y          = 1 << 7;
-        const FOG_ENABLED                = 1 << 8;
+        // const UNUSED                  = 1 << 8; // USE THIS IF YOU ADD A NEW FLAG
```

Comment:
Is the concern here that renumbering everything might conflict with other open PRs?

Reply:
I mostly just wasn't sure if that was the way to do it or not 😅